src/logic/safe/store/selectors/index.ts

```diff
@@ -208,6 +208,14 @@ export const safeModulesSelector = createSelector(safeSelector, safeFieldSelecto
 
 export const safeFeaturesEnabledSelector = createSelector(safeSelector, safeFieldSelector('featuresEnabled'))
 
+export const safeOwnersAddressesSelector = createSelector(safeOwnersSelector, (owners): string[] => {
+  if (!owners) {
+    return []
+  }
+
+  return owners?.map(({ address }) => address)?.toArray()
```

Comment:
It's not a good practice to perform the conversion from immutable.js objects to js data structures inside the selectors as it can possibly hurt the performance. Also, List and Array have almost 1:1 interface so I don't think it's truly necessary

https://stackoverflow.com/questions/39075570/when-using-redux-with-immutable-js-do-you-call-tojs-on-your-selector-or-do
https://redux.js.org/recipes/using-immutablejs-with-redux#your-selectors-should-return-immutablejs-objects

Reply:
fixed